addons/account/models/account_move.py

```diff
@@ -3413,6 +3414,13 @@ def action_switch_invoice_into_refund_credit_note(self):
                     ]
                 })
 
+    def action_move_switch_credit_note_to_invoice(self):
+        if any(move.move_type not in ('in_refund', 'out_refund') or move.posted_before for move in self):
+            raise ValidationError(_("This action isn't available for this document."))
+        for move in self:
+            move.name = False
+            move.move_type = move.move_type.replace('refund', 'invoice')
```

Comment:
Pretty sure that you'd need everything that is in `action_switch_invoice_into_refund_credit_note`
Maybe refactor with something like
```python
    def _switch_move_type(self, from_type, to_type):
        ...
```